validator/test/fixtures/constraints/resource_value_pattern_required/data.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPResourceValuePatternConstraintV1
+metadata:
+  name: resource_value_pattern_project_label
+spec:
+  severity: high
+  match:
+    gcp:
+      target: ["organization/*"]
+      exclude: [] # optional, default is no exclusions
+  parameters:
+    asset_type: "cloudresourcemanager.googleapis.com/Project"
+    field_name: "labels.billing_id"
+    required: true
```

Comment:
can we add 2 more test constraint fixtures: one with `required: false`, and the other with `required: true` and no pattern?

Reply:
Addressed with latest commits.